userguide/jails.rst

```diff
@@ -619,11 +620,23 @@ Click :guilabel:`NEXT` to view all jail
    |                        |              | addresses, enter the host MAC address and the jail MAC address separated by a space.                    |
    |                        |              |                                                                                                         |
    +------------------------+--------------+---------------------------------------------------------------------------------------------------------+
-   | nat_forwards           | checkbox     | `Network Address Translation (NAT) port forwarding <https://en.wikipedia.org/wiki/Port_forwarding>`__   |
-   |                        |              | options. Enabling on :guilabel:`NAT` jails adds :guilabel:`protocol`, :guilabel:`jail_port`, and        |
-   |                        |              | :guilabel:`host_port` port forwarding options. Additional port forwarding configurations can be created |
-   |                        |              | by clicking |ui-add|. Multiple jails can be set to use the same :guilabel:`jail_port`, but only         |
-   |                        |              | one running jail can use an individual port.                                                            |
+   | NAT Port Forwarding    | checkbox     | Configure the ports that allow remote access to the jail. Required to override the default settings or  |
```

Comment:
Reword first sentence:
```
Configure ports for remote access to the jail.
```
Non-specific "required".  Required how?  Some people will read "required to override" as "you must override". Better to rephrase as "Default settings (which ones?) cannot be overridden unless this is set..."